README.md

```diff
@@ -60,18 +60,27 @@ A repo where I stash info about how to make a slack bot using aws
     own feel free to change it in the code or make it configurable
   
 # Environment Variables
-  BOT_TOKEN is the slack apps token more info is (here:)[https://api.slack.com/bot-users]
-  
-  GOOGLE API KEY: You can generate one in the Google API Under Credentials
-  
-  GOOGLE_CUSTOM_SEARCH_KEY: Once you've enabled you Custom Search API check there it's the: Search Engine ID
 
   BING_SUBSCRIPTION_KEY: The Subscription Key from the Azure Account to use
 
   BING_CUSTOM_SEARCH_KEY: The Custom Search Key to use (Create in Azure under Bing Custom image search)
 
+  BOT_ID is the userid of the bot
```

Comment:
I recommend finding this when needed with https://api.slack.com/methods/auth.test 

I just tested it and got the info that would be needed.

Making it config is going to make standing up your own test stack more difficult to set up once we go to infrastructure as code (cloudformation).

Reply:
On a related note, i highly highly highly reccomend moving all keys and secrets to a aws secrets manager secret. You can make them each separate fields on a json map (secrets manager handles this extraordinarily well, making them seperate rows even in the ui https://docs.aws.amazon.com/secretsmanager/latest/userguide/intro.html) 

Moving to cloudformation will mean that manual steps would be needed to add environment variables (since every deployment would override them since you dont want to store secrets in github), while using a secrets manager requires setting once.

Reply:
Might be best to open a git issue. That seems out of bounds of this PR. I can work on it next. Seems easy to implement.

Reply:
Added: https://github.com/Splyth/slack_bot/issues/28